From 98187db50308883e313144f446d7f026f3f75db2 Mon Sep 17 00:00:00 2001
From: Andrei Cherechesu <andrei.cherechesu@nxp.com>
Date: Thu, 10 Feb 2022 16:06:59 +0200
Subject: [PATCH 40/43] dt-bindings: Remove all non-SCMI clock include files

Removed all clock dt-bindings include files which do not
contain any info related to SCMI clocks.

Issue: ALB-8352
Upstream-Status: Pending 

Signed-off-by: Andrei Cherechesu <andrei.cherechesu@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 include/dt-bindings/clock/s32g-clock.h        |  67 ---------
 include/dt-bindings/clock/s32g3-clock.h       |  16 --
 .../dt-bindings/clock/s32gen1-clock-freq.h    |  47 ------
 include/dt-bindings/clock/s32gen1-clock.h     | 140 ------------------
 include/dt-bindings/clock/s32r45-clock.h      |  34 -----
 5 files changed, 304 deletions(-)
 delete mode 100644 include/dt-bindings/clock/s32g-clock.h
 delete mode 100644 include/dt-bindings/clock/s32g3-clock.h
 delete mode 100644 include/dt-bindings/clock/s32gen1-clock-freq.h
 delete mode 100644 include/dt-bindings/clock/s32gen1-clock.h
 delete mode 100644 include/dt-bindings/clock/s32r45-clock.h

diff --git a/include/dt-bindings/clock/s32g-clock.h b/include/dt-bindings/clock/s32g-clock.h
deleted file mode 100644
index 3e7815a186..0000000000
--- a/include/dt-bindings/clock/s32g-clock.h
+++ /dev/null
@@ -1,67 +0,0 @@
-/* SPDX-License-Identifier: BSD-3-Clause */
-/*
- * Copyright 2020-2021 NXP
- */
-
-#ifndef __DT_BINDINGS_CLOCK_S32G_H
-#define __DT_BINDINGS_CLOCK_S32G_H
-
-#include <dt-bindings/clock/s32gen1-clock.h>
-
-#define S32G_CLK_ACCEL_PLL_PHI0		S32GEN1_CC_CLK(32)
-#define S32G_CLK_ACCEL_PLL_PHI1		S32GEN1_CC_CLK(33)
-#define S32G_CLK_SERDES0_LANE1_TX	S32GEN1_CC_CLK(46)
-#define S32G_CLK_SERDES0_LANE1_CDR	S32GEN1_CC_CLK(47)
-
-#define S32G_CLK_PFE_MAC0_EXT_TX	S32GEN1_CC_CLK(48)
-#define S32G_CLK_PFE_MAC0_EXT_RX	S32GEN1_CC_CLK(49)
-#define S32G_CLK_PFE_MAC0_EXT_REF	S32GEN1_CC_CLK(50)
-#define S32G_CLK_PFE_MAC1_EXT_TX	S32GEN1_CC_CLK(51)
-#define S32G_CLK_PFE_MAC1_EXT_RX	S32GEN1_CC_CLK(52)
-#define S32G_CLK_PFE_MAC1_EXT_REF	S32GEN1_CC_CLK(53)
-#define S32G_CLK_PFE_MAC2_EXT_TX	S32GEN1_CC_CLK(54)
-#define S32G_CLK_PFE_MAC2_EXT_RX	S32GEN1_CC_CLK(55)
-#define S32G_CLK_PFE_MAC2_EXT_REF	S32GEN1_CC_CLK(56)
-
-#define S32G_CLK_SERDES1_LANE0_TX	S32GEN1_CC_CLK(57)
-#define S32G_CLK_SERDES1_LANE0_CDR	S32GEN1_CC_CLK(58)
-
-#define S32G_CLK_PFE_MAC0_REF_DIV	S32GEN1_CC_CLK(59)
-#define S32G_CLK_PFE_MAC1_REF_DIV	S32GEN1_CC_CLK(60)
-#define S32G_CLK_PFE_MAC2_REF_DIV	S32GEN1_CC_CLK(61)
-
-#define S32G_CLK_SERDES1_LANE1_TX	S32GEN1_CC_CLK(62)
-#define S32G_CLK_SERDES1_LANE1_CDR	S32GEN1_CC_CLK(63)
-
-#define S32G_CLK(N)			S32GEN1_PLAT_CLK(N)
-#define S32G_CLK_INDEX(N)		S32GEN1_PLAT_ARRAY_INDEX(N)
-#define ARR_CLK(N)			S32G_CLK_INDEX(N)
-
-
-/* PFE_MAC0 */
-#define S32G_CLK_PFE_MAC0_TX_DIV	S32G_CLK(0)
-#define S32G_CLK_PFE_MAC0_RX		S32G_CLK(1)
-
-/* PFE_MAC1 */
-#define S32G_CLK_PFE_MAC1_TX		S32G_CLK(2)
-#define S32G_CLK_PFE_MAC1_RX		S32G_CLK(3)
-
-/* PFE_MAC2 */
-#define S32G_CLK_PFE_MAC2_TX		S32G_CLK(4)
-#define S32G_CLK_PFE_MAC2_RX		S32G_CLK(5)
-
-#define S32G_CLK_PFE_SYS		S32G_CLK(6)
-#define S32G_CLK_PFE_PE			S32G_CLK(7)
-
-#define S32G_CLK_MC_CGM2_MUX0		S32G_CLK(8)
-#define S32G_CLK_MC_CGM2_MUX1		S32G_CLK(9)
-#define S32G_CLK_MC_CGM2_MUX2		S32G_CLK(10)
-#define S32G_CLK_MC_CGM2_MUX3		S32G_CLK(11)
-#define S32G_CLK_MC_CGM2_MUX4		S32G_CLK(12)
-#define S32G_CLK_MC_CGM2_MUX5		S32G_CLK(13)
-#define S32G_CLK_MC_CGM2_MUX6		S32G_CLK(14)
-#define S32G_CLK_MC_CGM2_MUX7		S32G_CLK(15)
-#define S32G_CLK_MC_CGM2_MUX8		S32G_CLK(16)
-#define S32G_CLK_MC_CGM2_MUX9		S32G_CLK(17)
-
-#endif /* __DT_BINDINGS_CLOCK_S32G_H */
diff --git a/include/dt-bindings/clock/s32g3-clock.h b/include/dt-bindings/clock/s32g3-clock.h
deleted file mode 100644
index b5f89a49db..0000000000
--- a/include/dt-bindings/clock/s32g3-clock.h
+++ /dev/null
@@ -1,16 +0,0 @@
-/* SPDX-License-Identifier: BSD-3-Clause */
-/*
- * Copyright 2021 NXP
- */
-
-#ifndef __DT_BINDINGS_CLOCK_S32G3_H
-#define __DT_BINDINGS_CLOCK_S32G3_H
-
-#include <dt-bindings/clock/s32g-clock.h>
-
-#define S32G_CLK_MC_CGM6_MUX0		S32G_CLK(18)
-#define S32G_CLK_MC_CGM6_MUX1		S32G_CLK(19)
-#define S32G_CLK_MC_CGM6_MUX2		S32G_CLK(20)
-#define S32G_CLK_MC_CGM6_MUX3		S32G_CLK(21)
-
-#endif //__DT_BINDINGS_CLOCK_S32G3_H
diff --git a/include/dt-bindings/clock/s32gen1-clock-freq.h b/include/dt-bindings/clock/s32gen1-clock-freq.h
deleted file mode 100644
index 89e7d56d64..0000000000
--- a/include/dt-bindings/clock/s32gen1-clock-freq.h
+++ /dev/null
@@ -1,47 +0,0 @@
-/* SPDX-License-Identifier: BSD-3-Clause */
-/*
- * Copyright 2020-2021 NXP
- */
-#ifndef S32GEN1_CLOCK_FREQ_H
-#define S32GEN1_CLOCK_FREQ_H
-
-#include <config.h>
-
-#define MHZ					(1000000UL)
-
-/* Frequencies */
-#define S32GEN1_PERIPH_PLL_VCO_FREQ		(2000 * MHZ)
-#define S32GEN1_FXOSC_FREQ			(40 * MHZ)
-#define S32GEN1_FIRC_FREQ			(48 * MHZ)
-#define S32GEN1_LIN_BAUD_CLK_FREQ		(125 * MHZ)
-#define S32GEN1_DDR_PLL_VCO_FREQ		(1600 * MHZ)
-#define S32GEN1_DDR_FREQ			(800 * MHZ)
-
-#if defined(CONFIG_NXP_S32G2XX)
-#define S32GEN1_A53_MAX_FREQ			(1000 * MHZ)
-#define S32GEN1_ARM_PLL_VCO_MAX_FREQ		(2000 * MHZ)
-#define S32GEN1_ARM_PLL_PHI0_MAX_FREQ		(1000 * MHZ)
-#define S32GEN1_PERIPH_PLL_PHI0_MIN_FREQ	(100 * MHZ)
-#define S32GEN1_PERIPH_PLL_PHI2_MIN_FREQ	(40 * MHZ)
-#define S32GEN1_QSPI_MAX_FREQ			(200 * MHZ)
-
-#elif defined(CONFIG_NXP_S32R45)
-#define S32GEN1_A53_MAX_FREQ			(800 * MHZ)
-#define S32GEN1_ARM_PLL_VCO_MAX_FREQ		(1600 * MHZ)
-#define S32GEN1_ARM_PLL_PHI0_MAX_FREQ		(800 * MHZ)
-#define S32GEN1_PERIPH_PLL_PHI0_MIN_FREQ	(0 * MHZ)
-#define S32GEN1_PERIPH_PLL_PHI2_MIN_FREQ	(0 * MHZ)
-#define S32GEN1_QSPI_MAX_FREQ			(133333333)
-
-#elif defined(CONFIG_NXP_S32G3XX)
-#define S32GEN1_A53_MAX_FREQ			(1300 * MHZ)
-#define S32GEN1_ARM_PLL_VCO_MAX_FREQ		(2600 * MHZ)
-#define S32GEN1_ARM_PLL_PHI0_MAX_FREQ		(1300 * MHZ)
-#define S32GEN1_PERIPH_PLL_PHI0_MIN_FREQ	(100 * MHZ)
-#define S32GEN1_PERIPH_PLL_PHI2_MIN_FREQ	(40 * MHZ)
-#define S32GEN1_QSPI_MAX_FREQ			(200 * MHZ)
-#else
-#error Undefined platform
-#endif
-
-#endif
diff --git a/include/dt-bindings/clock/s32gen1-clock.h b/include/dt-bindings/clock/s32gen1-clock.h
deleted file mode 100644
index 65ad70d99a..0000000000
--- a/include/dt-bindings/clock/s32gen1-clock.h
+++ /dev/null
@@ -1,140 +0,0 @@
-/* SPDX-License-Identifier: BSD-3-Clause */
-/*
- * Copyright 2020 NXP
- */
-#ifndef __DT_BINDINGS_CLOCK_S32GEN1_H
-#define __DT_BINDINGS_CLOCK_S32GEN1_H
-
-#include <dt-bindings/clock/s32gen1-clock-freq.h>
-
-#define S32GEN1_CLK_ID_BASE			10000
-#define S32GEN1_CC_CLK(N)			((N) + S32GEN1_CLK_ID_BASE)
-#define S32GEN1_CC_ARRAY_INDEX(N)		((N) - S32GEN1_CLK_ID_BASE)
-#define CC_ARR_CLK(N)				S32GEN1_CC_ARRAY_INDEX(N)
-
-/* Clock source / Clock selector index */
-#define S32GEN1_CLK_FIRC			S32GEN1_CC_CLK(0)
-#define S32GEN1_CLK_SIRC			S32GEN1_CC_CLK(1)
-#define S32GEN1_CLK_FXOSC			S32GEN1_CC_CLK(2)
-#define S32GEN1_CLK_ARM_PLL_PHI0		S32GEN1_CC_CLK(4)
-#define S32GEN1_CLK_ARM_PLL_PHI1		S32GEN1_CC_CLK(5)
-#define S32GEN1_CLK_ARM_PLL_PHI2		S32GEN1_CC_CLK(6)
-#define S32GEN1_CLK_ARM_PLL_PHI3		S32GEN1_CC_CLK(7)
-#define S32GEN1_CLK_ARM_PLL_PHI4		S32GEN1_CC_CLK(8)
-#define S32GEN1_CLK_ARM_PLL_PHI5		S32GEN1_CC_CLK(9)
-#define S32GEN1_CLK_ARM_PLL_PHI6		S32GEN1_CC_CLK(10)
-#define S32GEN1_CLK_ARM_PLL_PHI7		S32GEN1_CC_CLK(11)
-#define S32GEN1_CLK_ARM_PLL_DFS1		S32GEN1_CC_CLK(12)
-#define S32GEN1_CLK_ARM_PLL_DFS2		S32GEN1_CC_CLK(13)
-#define S32GEN1_CLK_ARM_PLL_DFS3		S32GEN1_CC_CLK(14)
-#define S32GEN1_CLK_ARM_PLL_DFS4		S32GEN1_CC_CLK(15)
-#define S32GEN1_CLK_ARM_PLL_DFS5		S32GEN1_CC_CLK(16)
-#define S32GEN1_CLK_ARM_PLL_DFS6		S32GEN1_CC_CLK(17)
-#define S32GEN1_CLK_PERIPH_PLL_PHI0		S32GEN1_CC_CLK(18)
-#define S32GEN1_CLK_PERIPH_PLL_PHI1		S32GEN1_CC_CLK(19)
-#define S32GEN1_CLK_PERIPH_PLL_PHI2		S32GEN1_CC_CLK(20)
-#define S32GEN1_CLK_PERIPH_PLL_PHI3		S32GEN1_CC_CLK(21)
-#define S32GEN1_CLK_PERIPH_PLL_PHI4		S32GEN1_CC_CLK(22)
-#define S32GEN1_CLK_PERIPH_PLL_PHI5		S32GEN1_CC_CLK(23)
-#define S32GEN1_CLK_PERIPH_PLL_PHI6		S32GEN1_CC_CLK(24)
-#define S32GEN1_CLK_PERIPH_PLL_PHI7		S32GEN1_CC_CLK(25)
-#define S32GEN1_CLK_PERIPH_PLL_DFS1		S32GEN1_CC_CLK(26)
-#define S32GEN1_CLK_PERIPH_PLL_DFS2		S32GEN1_CC_CLK(27)
-#define S32GEN1_CLK_PERIPH_PLL_DFS3		S32GEN1_CC_CLK(28)
-#define S32GEN1_CLK_PERIPH_PLL_DFS4		S32GEN1_CC_CLK(29)
-#define S32GEN1_CLK_PERIPH_PLL_DFS5		S32GEN1_CC_CLK(30)
-#define S32GEN1_CLK_PERIPH_PLL_DFS6		S32GEN1_CC_CLK(31)
-#define S32GEN1_CLK_FTM0_EXT_REF		S32GEN1_CC_CLK(34)
-#define S32GEN1_CLK_FTM1_EXT_REF		S32GEN1_CC_CLK(35)
-#define S32GEN1_CLK_DDR_PLL_PHI0		S32GEN1_CC_CLK(36)
-#define S32GEN1_CLK_GMAC0_EXT_TX		S32GEN1_CC_CLK(37)
-#define S32GEN1_CLK_GMAC0_EXT_RX		S32GEN1_CC_CLK(38)
-#define S32GEN1_CLK_GMAC0_EXT_REF		S32GEN1_CC_CLK(39)
-#define S32GEN1_CLK_SERDES0_LANE0_TX		S32GEN1_CC_CLK(40)
-#define S32GEN1_CLK_SERDES0_LANE0_CDR		S32GEN1_CC_CLK(41)
-#define S32GEN1_CLK_GMAC0_EXT_TS		S32GEN1_CC_CLK(44)
-#define S32GEN1_CLK_GMAC0_REF_DIV		S32GEN1_CC_CLK(45)
-
-/* Total number of clocks sources listed in RM */
-#define S32GEN1_CLK_NUM_PROVIDERS		(64)
-#define S32GEN1_PERIPH_CLK(N)			(S32GEN1_CLK_NUM_PROVIDERS + \
-						 (N) + S32GEN1_CLK_ID_BASE)
-#define S32GEN1_PLAT_CLK_ID_BASE		(20000)
-#define S32GEN1_PLAT_CLK(N)			(S32GEN1_PLAT_CLK_ID_BASE + (N))
-#define S32GEN1_PLAT_ARRAY_INDEX(N)		((N) - S32GEN1_PLAT_CLK_ID_BASE)
-
-/* Software defined clocks */
-/* ARM PLL */
-#define S32GEN1_CLK_ARM_PLL_MUX			S32GEN1_PERIPH_CLK(0)
-#define S32GEN1_CLK_ARM_PLL_VCO			S32GEN1_PERIPH_CLK(1)
-
-/* ARM CGM1 clocks */
-#define S32GEN1_CLK_MC_CGM1_MUX0		S32GEN1_PERIPH_CLK(2)
-#define S32GEN1_CLK_A53_CORE			S32GEN1_PERIPH_CLK(3)
-#define S32GEN1_CLK_A53_CORE_DIV2		S32GEN1_PERIPH_CLK(4)
-#define S32GEN1_CLK_A53_CORE_DIV10		S32GEN1_PERIPH_CLK(5)
-
-/* ARM CGM0 clocks */
-#define S32GEN1_CLK_MC_CGM0_MUX0		S32GEN1_PERIPH_CLK(6)
-#define S32GEN1_CLK_XBAR_2X			S32GEN1_PERIPH_CLK(7)
-#define S32GEN1_CLK_XBAR			S32GEN1_PERIPH_CLK(8)
-#define S32GEN1_CLK_XBAR_DIV2			S32GEN1_PERIPH_CLK(9)
-#define S32GEN1_CLK_XBAR_DIV3			S32GEN1_PERIPH_CLK(10)
-#define S32GEN1_CLK_XBAR_DIV4			S32GEN1_PERIPH_CLK(11)
-#define S32GEN1_CLK_XBAR_DIV6			S32GEN1_PERIPH_CLK(12)
-
-/* PERIPH PLL */
-#define S32GEN1_CLK_PERIPH_PLL_MUX		S32GEN1_PERIPH_CLK(13)
-#define S32GEN1_CLK_PERIPH_PLL_VCO		S32GEN1_PERIPH_CLK(14)
-
-/* PERIPH CGM0 clocks */
-#define S32GEN1_CLK_SERDES_REF			S32GEN1_PERIPH_CLK(15)
-#define S32GEN1_CLK_MC_CGM0_MUX3		S32GEN1_PERIPH_CLK(16)
-#define S32GEN1_CLK_PER				S32GEN1_PERIPH_CLK(17)
-#define S32GEN1_CLK_MC_CGM0_MUX4		S32GEN1_PERIPH_CLK(18)
-#define S32GEN1_CLK_FTM0_REF			S32GEN1_PERIPH_CLK(19)
-#define S32GEN1_CLK_MC_CGM0_MUX5		S32GEN1_PERIPH_CLK(20)
-#define S32GEN1_CLK_FTM1_REF			S32GEN1_PERIPH_CLK(21)
-#define S32GEN1_CLK_MC_CGM0_MUX6		S32GEN1_PERIPH_CLK(22)
-#define S32GEN1_CLK_FLEXRAY_PE			S32GEN1_PERIPH_CLK(23)
-#define S32GEN1_CLK_MC_CGM0_MUX7		S32GEN1_PERIPH_CLK(24)
-#define S32GEN1_CLK_CAN_PE			S32GEN1_PERIPH_CLK(25)
-#define S32GEN1_CLK_MC_CGM0_MUX8		S32GEN1_PERIPH_CLK(26)
-#define S32GEN1_CLK_LIN_BAUD			S32GEN1_PERIPH_CLK(27)
-#define S32GEN1_CLK_LINFLEXD			S32GEN1_PERIPH_CLK(28)
-#define S32GEN1_CLK_MC_CGM0_MUX1		S32GEN1_PERIPH_CLK(29)
-#define S32GEN1_CLK_MC_CGM0_MUX2		S32GEN1_PERIPH_CLK(30)
-#define S32GEN1_CLK_MC_CGM0_MUX9		S32GEN1_PERIPH_CLK(31)
-#define S32GEN1_CLK_MC_CGM0_MUX10		S32GEN1_PERIPH_CLK(32)
-#define S32GEN1_CLK_MC_CGM0_MUX11		S32GEN1_PERIPH_CLK(33)
-#define S32GEN1_CLK_MC_CGM0_MUX12		S32GEN1_PERIPH_CLK(34)
-#define S32GEN1_CLK_MC_CGM0_MUX13		S32GEN1_PERIPH_CLK(35)
-#define S32GEN1_CLK_MC_CGM0_MUX14		S32GEN1_PERIPH_CLK(36)
-#define S32GEN1_CLK_MC_CGM0_MUX15		S32GEN1_PERIPH_CLK(37)
-#define S32GEN1_CLK_MC_CGM0_MUX16		S32GEN1_PERIPH_CLK(38)
-#define S32GEN1_CLK_SPI				S32GEN1_PERIPH_CLK(39)
-#define S32GEN1_CLK_QSPI_2X			S32GEN1_PERIPH_CLK(40)
-#define S32GEN1_CLK_QSPI			S32GEN1_PERIPH_CLK(41)
-#define S32GEN1_CLK_SDHC			S32GEN1_PERIPH_CLK(42)
-
-/* DDR PLL */
-#define S32GEN1_CLK_DDR_PLL_MUX			S32GEN1_PERIPH_CLK(43)
-#define S32GEN1_CLK_DDR_PLL_VCO			S32GEN1_PERIPH_CLK(44)
-#define S32GEN1_CLK_MC_CGM5_MUX0		S32GEN1_PERIPH_CLK(45)
-#define S32GEN1_CLK_DDR				S32GEN1_PERIPH_CLK(46)
-
-/* ACCEL PLL */
-#define S32GEN1_CLK_ACCEL_PLL_MUX		S32GEN1_PERIPH_CLK(47)
-#define S32GEN1_CLK_ACCEL_PLL_VCO		S32GEN1_PERIPH_CLK(48)
-
-/* CLKOUT */
-#define S32GEN1_CLK_CLKOUT0			S32GEN1_PERIPH_CLK(49)
-#define S32GEN1_CLK_CLKOUT1			S32GEN1_PERIPH_CLK(50)
-
-/* GMAC0 */
-#define S32GEN1_CLK_GMAC0_TS			S32GEN1_PERIPH_CLK(51)
-#define S32GEN1_CLK_GMAC0_TX			S32GEN1_PERIPH_CLK(52)
-#define S32GEN1_CLK_GMAC0_RX			S32GEN1_PERIPH_CLK(53)
-#define S32GEN1_CLK_GMAC0_REF			S32GEN1_PERIPH_CLK(54)
-
-#endif
diff --git a/include/dt-bindings/clock/s32r45-clock.h b/include/dt-bindings/clock/s32r45-clock.h
deleted file mode 100644
index 5b161c5ed7..0000000000
--- a/include/dt-bindings/clock/s32r45-clock.h
+++ /dev/null
@@ -1,34 +0,0 @@
-/* SPDX-License-Identifier: BSD-3-Clause */
-/*
- * Copyright 2020-2021 NXP
- */
-
-#ifndef __DT_BINDINGS_CLOCK_S32R45_CLK_H
-#define __DT_BINDINGS_CLOCK_S32R45_CLK_H
-
-#include <dt-bindings/clock/s32gen1-clock.h>
-
-#define S32R45_CLK_GMAC1_REF_DIV		S32GEN1_CC_CLK(54)
-#define S32R45_CLK_GMAC1_EXT_TX			S32GEN1_CC_CLK(56)
-#define S32R45_CLK_GMAC1_EXT_RX			S32GEN1_CC_CLK(57)
-#define S32R45_CLK_GMAC1_EXT_REF		S32GEN1_CC_CLK(58)
-#define S32R45_CLK_SERDES1_LANE0_TX		S32GEN1_CC_CLK(59)
-#define S32R45_CLK_SERDES1_LANE0_CDR		S32GEN1_CC_CLK(60)
-#define S32R45_CLK_ACCEL_PLL_PHI0		S32GEN1_CC_CLK(62)
-#define S32R45_CLK_ARM_PLL_DFS4_2		S32GEN1_CC_CLK(63)
-
-#define S32R45_CLK(N)			S32GEN1_PLAT_CLK(N)
-#define S32R45_CLK_INDEX(N)		S32GEN1_PLAT_ARRAY_INDEX(N)
-
-#define S32R45_CLK_MC_CGM2_MUX0		S32R45_CLK(0)
-#define S32R45_CLK_ACCEL3		S32R45_CLK(1)
-#define S32R45_CLK_ACCEL3_DIV3		S32R45_CLK(2)
-#define S32R45_CLK_MC_CGM2_MUX1		S32R45_CLK(3)
-#define S32R45_CLK_ACCEL4		S32R45_CLK(4)
-#define S32R45_CLK_MC_CGM2_MUX2		S32R45_CLK(5)
-#define S32R45_CLK_GMAC1_TX		S32R45_CLK(6)
-#define S32R45_CLK_MC_CGM2_MUX3		S32R45_CLK(7)
-#define S32R45_CLK_MC_CGM2_MUX4		S32R45_CLK(9)
-#define S32R45_CLK_GMAC1_RX		S32R45_CLK(10)
-
-#endif
-- 
2.17.1

